### PR TITLE
Make own map box with toggle

### PR DESCRIPTION
**Pull request by fridasolheim**
Map is now visible in compact mode independently of whether there are any scooters or not/if scooters are activated. Can now also show only bikes, only scooters and/or only stop places. Map has its own box in the admin panel, and can be toggled on/off.

### CODE REVIEW

**Comment by fridasolheim** [comment]
Screenshot 2021-06-25 at 15 51 44

**Comment by draperunner** [comment]
Fixes #332 :)

File: src/dashboards/Compact/index.tsx [hunk 1]
```diff
@@ -23,6 +23,7 @@ import BikeTile from './BikeTile'
 import MapTile from './MapTile'
 
 import './styles.scss'
+//import { MapController } from 'react-map-gl'
```
Comment (draperunner) [inline src/dashboards/Compact/index.tsx:26]: Du kan godt fjerne utkommentert kode.

File: src/dashboards/Compact/index.tsx [hunk 2]
```diff
@@ -145,12 +152,14 @@ const EnturDashboard = ({ history }: Props): JSX.Element => {
                     ) : (
                         []
                     )}
-                    {scooters?.length ? (
+                    {scooters?.length ||
```
Comment (draperunner) [inline src/dashboards/Compact/index.tsx:155]: Kanskje dette kunne blitt dratt ut i ein `hasData`-variabel som kan brukast både her og i `mapCol`?

```
const hasData = Boolean(bikeRentalStations?.length ||
        scooters?.length ||
        stopPlacesWithDepartures?.length)
        
const mapCol = hasData ? 1 : 0
```

File: src/settings/index.ts [hunk 3]
```diff
@@ -19,7 +19,7 @@ import {
 } from './UrlStorage'
 import { persist as persistToFirebase, FieldTypes } from './FirestoreStorage'
 
-export type Mode = 'bysykkel' | 'kollektiv' | 'sparkesykkel'
+export type Mode = 'bysykkel' | 'kollektiv' | 'sparkesykkel' | 'kartfunksjon'
```
Comment (draperunner) [inline src/settings/index.ts:22]: `'kartfunksjon'` vart vel ikkje tatt i bruk likevel, så den kan bort.

**Review by draperunner — CHANGES_REQUESTED** [review]

**Review by draperunner — APPROVED** [review]

**Pull request merged**